docs/sources/reference/components/loki/loki.secretfilter.md

```diff
@@ -51,7 +51,7 @@ The `types` argument is a map of secret types to look for. The values are used a
 
 The `redact_with` argument is a string that can use variables such as `$SECRET_NAME` (replaced with the matching secret type) and `$SECRET_HASH`(replaced with the sha1 hash of the secret).
 
-The `exclude_generic` argument is a boolean that excludes the generic API key rule in the Gitleaks configuration file if set to `true`.
+The `include_generic` argument is a boolean that includes the generic API key rule in the Gitleaks configuration file if set to `true`. It is disabled by default because it can generate false positives.
```

Comment:
should not remove it completely if it's not good?

Reply:
It can be really useful! But because it's a generic secret-detection regex, it's definitely not perfect and some logs will have a format that sometimes triggers it (especially when they contain keywords like `secret`).

My opinion is that it should be disabled by default and enabled explicitly when the user has confidence that their log format doesn't generate false positives (or they excluded them), so that it can start picking up real secrets that weren't caught by the more specialised rules.

So in short, it can definitely be useful but requires a bit more work from the user.